test/api.jl

```diff
@@ -202,3 +203,13 @@ end
         @test RAI.exec(ctx, "engine", "db", "2+2") isa RAI.TransactionResponse
     end
 end
+
+@testset "hide client secrets in repl" begin
+    access_token = AccessToken("abc_token", "run:transaction", 3600, DateTime("2022-08-12T17:49:51.365"))
+    creds = ClientCredentials("client_id", "xyz_client_secret", "https://login.relationalai.com/oauth/token")
+    creds.access_token = access_token
+
+    io = IOBuffer()
+    show(io, creds)
+    @test String(take!(io)) === "(client_id, xyz..., (abc..., run:transaction, 3600, 2022-08-12T17:49:51.365), https://login.relationalai.com/oauth/token)"
+end
```

Comment:
missing endline
(There's a setting to enable always adding an endline in VSCode if you use it, which is nice):
![unnamed](https://user-images.githubusercontent.com/1582097/184994868-9f381d2e-f01a-4eda-b014-0c58e82e13d7.png)